web-ui/src/main/resources/catalog/locales/fr-search.json

```diff
@@ -546,5 +546,6 @@
     "fieldEmailNotValid": "Un email valide est requis",
     "addLayerPrefix": "Ajouter une couche",
     "addLayerPostfix": "à la carte",
-    "submit": "Soumettre"
+    "submit": "Soumettre",
+    "additionalLinks": "Additional Links"
```

Comment:
This is a new field that is part of 4.8 and applied with this PR https://github.com/geonetwork/core-geonetwork/pull/7802.

So it would not be in the 4.2.12 release so only would get translated in 4.2.13.

It does not hurt to add it so I will approve even with this change.